libs/components/i18n/schematics/ng-generate/lib-resources-module/files/__name@dasherize__-resources.module.ts.template

```diff
@@ -23,7 +22,10 @@ import {
 SkyLibResourcesService.addResources(RESOURCES);
 
 export class <%= classify(name) %>ResourcesProvider implements SkyLibResourcesProvider {
-  public getString(localeInfo: SkyAppLocaleInfo, name: string): string | undefined {
+  public getString(
+    localeInfo: SkyAppLocaleInfo,
+    name: string,
+  ): string | undefined {
```

Comment:
These changes are necessary to satisfy the "check resources" job after code formatting runs.